### ISSUE TITLE
[iOS/Android] Customize attribution position

### ISSUE THREAD

**Issue by karlguillotte** [nitaliano/react-native-mapbox-gl#961]
Hi, 

The attribution is positioned on the bottom left on Android and positioned on the bottom right on iOS. It is a little detail, but I put some buttons over my map and they interfere with that inconsistent behavior. 

Thanks!

**Comment by nitaliano** [comment]
fair enough I'll look into seeing how much control the Android SDK gives us with that icon

**Comment by derekblank** [comment]
I also noticed that when `logoEnabled={false}` on `MapView`, the attribution icon does not adjust to the corner.

Default:
![logoenabled](https://user-images.githubusercontent.com/643285/37054629-3ae811e6-2134-11e8-9a00-16e33a635cac.png)

`logoEnabled={false}`:
![logoenabledfalse](https://user-images.githubusercontent.com/643285/37054573-19c881c6-2134-11e8-87ef-93d1e9821b81.png)

react-native-mapbox-gl v6.1.0
Android v8.0.0

**Comment by karlguillotte** [comment]
@nitaliano Sorry to bother you, wondering if you have had a chance to look at this? Thanks.

**Comment by Foosballfan** [comment]
I've taken a look through the docs and it looks as though this could be controlled using [setAttributionMargins ](https://www.mapbox.com/android-docs/api/map-sdk/6.7.2/com/mapbox/mapboxsdk/maps/UiSettings.html#setAttributionMargins-int-int-int-int-).

A new method can be created in the [java](https://github.com/mapbox/react-native-mapbox-gl/blob/4aac2ff60a6e5a065dcf5b670c0342cb69516cf1/android/rctmgl/src/main/java/com/mapbox/rctmgl/components/mapview/RCTMGLMapView.java) which can then be triggered in the JS using _runNativeCommand,  similar to the way that getPointInView() is set up.

Assuming the attribution is currently controlled by it's margins (may be incorrect) then we would have full control of the attribute positioning from within React Native.

**Comment by VinzGhyz** [comment]
Same issue here, would love to be able to update attribution positioning as our design renders it untappable. Anyone managed to move it?

**Comment by kristfal** [comment]
Hey,

happy to accept PRs for this. 

A potential solution: Completely disable watermarks on the native side and render these (using correct assets and specs provided by Mapbox) as RN views. That will allow us more control and easier cross-platform maintenance.
